scripts/dockerfiles/readiness-probe.sh

```diff
@@ -40,14 +41,17 @@ if test -n "${FISSILE_ACTIVE_PASSIVE_PROBE:-}" ; then
     trap update_readiness EXIT
 fi
 
+if [ ! -r /etc/monitrc ] ; then
+    echo "Waiting for monit to start"
```

Comment:
Silence the misleading error about `/etc/monitrc` not existing (that makes it seem like something has really gone wrong, when it's just not finished running configgin yet)